### ISSUE TITLE
[Accessibility]: Global nav bar: Usability issues at 320px

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#35117]
### Audit type

Mobile device navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34411

### Problem description

<img width="489" alt="image" src="https://user-images.githubusercontent.com/9516420/167415925-f1cd6f67-0c49-4315-9bb5-137de9c08f39.png">

Difficult to select "Login" or "Get started"

### Expected behavior

Something like:

<img width="514" alt="image" src="https://user-images.githubusercontent.com/9516420/167416096-59f5642d-7d64-4031-bd3b-57eaa6a44c0c.png">

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
Confirmed that this has already been fixed
